daemon/api_sideload_n_try.go

```diff
@@ -190,6 +191,14 @@ func sideloadOrTrySnap(c *Command, body io.ReadCloser, boundary string, user *au
 	st.Lock()
 	defer st.Unlock()
 
+	if err := verifySnapdIsNotRefreshing(st); err != nil {
```

Comment:
we never do this thing at this level, they are done via snapstate.CheckChangeConflict*, see conflict.go there